modules/ding_place2book/ding_place2book.module

```diff
@@ -1049,3 +1124,12 @@ function ding_place2book_theme() {
   );
   return $items;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ding_place2book_preprocess_node(&$variables) {
```

Comment:
A single comment about why this "access" false is here would be nice.

Reply:
See https://github.com/ding2/ding2/pull/1010/commits/f0fe3bfb10e7bde6e501b55c1028a54d668458f1#diff-8b99672f6dd58da79fd6b49feeca23adR1133

Reply:
👍